core/src/main/scala/kafka/tools/ConsoleConsumer.scala

```diff
@@ -38,7 +38,13 @@ object ConsoleConsumer extends Logging {
 
   def main(args: Array[String]) {
     val conf = new ConsumerConfig(args)
-    run(conf)
+    try {
+      run(conf)
+    } catch {
+      case e: Throwable =>
+        error("Error running consumer: ", e)
+    }
+    System.exit(0);
```

Comment:
Yes, that would be better as we would return a non-zero code which some command-line tools may rely on. I would also suggest removing the braces for the `try` as they are not needed for a single expression `try`.